### PR TITLE
Enable updating the password of a user.

### PR DESCRIPTION
**Pull request by pspoerri**
`UserUpdate` does not have a `hashed_password`.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/backend/app/app/crud/crud_user.py [hunk 1]
```diff
@@ -24,6 +24,11 @@ def create(self, db_session: Session, *, obj_in: UserCreate) -> User:
         db_session.refresh(db_obj)
         return db_obj
 
+    def update(self, db_session: Session, *, db_obj: ModelType, obj_in: UpdateSchemaType) -> ModelType:
+        if obj_in.password:
+            db_obj.hashed_password = get_password_hash(obj_in.password)
```
Comment (pspoerri) [inline {{cookiecutter.project_slug}}/backend/app/app/crud/crud_user.py]: I am not sure if this is the correct place to update the password.

Comment (ebreton) [reply {{cookiecutter.project_slug}}/backend/app/app/crud/crud_user.py]: You probably want to use the `reset_password` method in [login.py](https://github.com/tiangolo/full-stack-fastapi-postgresql/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/api/api_v1/endpoints/login.py)

**Comment by laurentS** [comment]
I was about to open almost the same PR to fix #109. I've put a small diff in there that you might want to add to your PR to fix the test as well :)

File: {{cookiecutter.project_slug}}/backend/app/app/crud/crud_user.py (hunk 1 above)
Comment (pspoerri) [reply {{cookiecutter.project_slug}}/backend/app/app/crud/crud_user.py]: > You probably want to use the `reset_password` method in [login.py](https://github.com/tiangolo/full-stack-fastapi-postgresql/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/api/api_v1/endpoints/login.py)

Shouldn't it be the other way around, e.g. the endpoint/login.py using CRUDUser?

**Comment by tiangolo** [comment]
Thanks for your contribution! :rocket:

I think this was solved/included in https://github.com/tiangolo/full-stack-fastapi-postgresql/pull/106 , right?

**Comment by pspoerri** [comment]
Yes, looks like it. 😅

**Comment by tiangolo** [comment]
Thanks for reporting back and closing it :+1:

**Pull request closed without merging**